src/components/cody/HowCodyWorks.tsx

```diff
@@ -5,50 +5,55 @@ import classNames from 'classnames'
 import { ContentSection } from '..'
 import { DevPlatformsSection } from '../Enterprise/DevPlatformsSection'
 
-export const HowCodyWorks: FunctionComponent<{ isLight: boolean }> = ({ isLight = false }) => (
+export const HowCodyWorks: FunctionComponent<{ isLight: boolean; isVariant: boolean }> = ({
+    isLight = false,
+    isVariant,
+}) => (
     <ContentSection
-        parentClassName="!py-0 !px-0"
+        parentClassName="!py-16 md:!py-28 "
         className={classNames(
-            'flex h-auto max-w-[1280px] flex-col justify-between overflow-hidden border-y px-6 px-0 md:my-24 md:h-[414px] md:flex-row md:justify-between md:gap-2 md:rounded-2xl md:border-1 md:pl-16 md:pr-[82.04px]',
-            { 'border-[#343A4D] bg-violet-700': !isLight, 'hidden border-gray-200 bg-[#FFF] md:flex': isLight }
+            'flex h-auto  max-w-[1280px] flex-col justify-between overflow-hidden rounded-2xl border px-6 px-0  md:my-0 md:flex-row md:justify-between md:gap-2 md:border-1 md:pl-16 md:pr-[82.04px]',
+            {
+                'border-[#343A4D] bg-violet-700': !isLight,
+                'border-gray-200 bg-[#FFF] md:flex': isLight,
+                'pb-[92px] md:h-[414px]': !isVariant,
+                'flex !items-center pb-24 md:h-[458px] md:pb-0': isVariant,
+            }
         )}
     >
         <div
```

Comment:
Could we actually have the heading + sub content centered vertically in the container instead? The big whitespace underneath it makes it feel incomplete. 

<img width="1355" alt="Screenshot 2024-09-25 at 8 49 34 AM" src="https://github.com/user-attachments/assets/c3dc9212-e1f4-44f5-8827-303bd2e3bafb">